x/gov/keeper/tally.go

```diff
@@ -118,6 +118,13 @@ func (keeper Keeper) Tally(ctx sdk.Context, proposal types.Proposal) (passes boo
 		return true, false, tallyResults
 	}
 
+	// if the proposal is expedited and less than expedited_quorum_threshold, the proposal fails
+	if proposal.IsExpedited {
+		if percentVoting.LT(tallyParams.ExpeditedQuorumThreshold) {
+			return false, true, tallyResults
+		}
+	}
```

Comment:
Please consider to instead create a method similar to this and use it on line 100
https://github.com/osmosis-labs/cosmos-sdk/blob/17194967ba663f8e4a0455860174338bafec30a5/x/gov/types/params.go#L152

This would minimize the changes needed and make the abstraction consistent with the threshold